azure-pipelines.yml

```diff
@@ -0,0 +1,128 @@
+name: '6.1$(rev:.r)'
```

Comment:
I'm not opposed to switching away from the classic pipelines, however I might need to work through this configuration with you all to make sure I understand how the dev -> prod workflow will work.

Reply:
It is quite easy. If you want we can have a zoom call and discuss this ;)

Reply:
@JoostVoskuil That would be helpful. I'm going to be tied up this week for the most part at a conference, would you be available the next week to review?

Reply:
Hi @ejohn20 I have send you an email ;)